Paa1006-Assignment4/src/paa1006/assignment4/Paa1006Assignment4.java

```diff
@@ -0,0 +1,144 @@
+import java.util.Date;      // Needed for Date and Time
+import java.text.DateFormat;    // Needed for Date and Time
+import java.text.SimpleDateFormat;  // Needed for Date and Time
+import java.util.Calendar;      // Needed for Date and Time
+import java.util.concurrent.TimeUnit;
+
+public class Paa1006Assignment4 {
+
+    public static void main(String[] args) {
+        
+    // Decale variable to test for Do While loop
+    Boolean exitP = false;      
+    do{
+        // Declare variables
+        int menuNumber;
+        Scanner keyboard = new Scanner(System.in);
+        
+        // Show User Menu and get input
+        System.out.println("--------------------------");
+        System.out.println("1. Value of a String");
+        System.out.println("2. Current Date and Time");
+        System.out.println("3. Programmer's Name");
+        System.out.println("4. IDE Used to to Make This");
+        System.out.println("5. Programmer's Github Username");
+        System.out.println("6. Exit Application");
+        System.out.print("Enter a number to select a menu option: ");
+        menuNumber = keyboard.nextInt();
+        System.out.println("You selected " + menuNumber);
+        System.out.println("--------------------------");
+     
+        // Determine the number entered
+        switch (menuNumber)
+        {
+            case 1:
+                // Take a word from user then send it to Method
+                Scanner keyboard2 = new Scanner(System.in);
+                String word;
+                System.out.print("Enter a string: ");
+                word = keyboard2.next();
+                
+                // Call method to get value of string
+               ValueOfString(word);
+               break;
+            case 2:
+                // Call method to Display Date and time
+                DisplayDateTime();
+                break;
+            case 3:
+                // Call method to display name
+                DisplayName();
+                break;
+            case 4:
+                // Call method to display IDE
+                DisplayIDE();
+                break;
+            case 5:
+                // Call method to display Github user name
+                DisplayGithub();
+                break;
+            case 6:
+                // Call method to display close program message
+                exitP = CloseProgram();
+                break;
+            default:
+                // Call method to display wrong input
+                WrongInput();
+        }
+    }while(exitP == false);
+}
+    
+    /**
+     * Method which takes a string, and finds its integer value
+     * @param word String
+     */
+    public static void ValueOfString(String word){
+        // Initialize a counter for value of string
+        int valueTotal = 0;
+        
+        // Loop to add up chars in the string
+        for(int i = 0;i < word.length();i++){
+          valueTotal += word.charAt(i);
+        }
```

Comment:
I like that you went back through and updated this word value calculator assignment to the most recent version we created. I did not think to do this with mine.